doc/nrf/ug_thingy91.rst

```diff
@@ -0,0 +1,262 @@
+
+Operating modes
+===============
+
+Thingy:91 contains RGB indicator LEDs, which indicate the operating state of the device as described in :ref:`operating states of Thingy:91<thingy91_operating_states>`.
+
+GPS
+===
+
+Thingy:91 has GPS, which if activated, allows the device to be located globally using GPS signals.
+See :ref:`Button SW3 on Thingy:91<asset_tracker_user_interface>` for information on the activation of GPS button on Thingy:91.
+
+LTE Band Lock
+=============
+
+The modem within Thingy:91 can be configured to use specific LTE bands by using the band lock AT command.
+See `nRF9160 Band lock`_ and `band lock section in the AT Commands reference document`_ for additional information.
+The pre-flashed firmware configures the modem to use the bands currently certified on the Thingy:91 hardware.
+The bands to be enabled can be configured when building the firmware.
+
+LTE-M / NB-IoT switching
+========================
+
+Thingy:91 has a multimode modem, which enables it to support automatic switching between LTE-M and NB-IoT.
+A built-in parameter in the Thingy:91 firmware determines whether the modem first attempts to connect in LTE-M or NB-IoT mode.
+If the modem fails to connect using this preferred mode within the default timeout period (10 minutes), the modem switches to the other mode.
+
+Modem firmware
+==============
+
+The functionality of the multimode modem of Thingy:91 is based on its firmware which is available as a pre-compiled binary.
+You can download the modem firmware from the `Thingy:91 product website (downloads)`_.
+The downloadable zip file contains the full firmware image and the firmware image files for updating the firmware.
+
+.. _precompiled_fw:
+
+Programming pre-compiled firmware images
+****************************************
+
+Pre-compiled firmware image files are useful in the following scenarios:
+
+* Restoring the firmware to its initial image
+
+* Updating the application firmware to a newly released version
+
+* Updating the modem firmware
+
+Downloading pre-compiled firmware images
+========================================
+
+To obtain pre-compiled firmware images for firmware update, perform the following steps:
+
+	1. Go to the `Thingy:91 product description`_ page and under the **Downloads** tab, navigate to **Precompiled firmware**.
+	#. Download and extract the latest Thingy:91 firmware package.
+	#. Check the ``CONTENTS.txt`` file in the extracted folder for the location and names of the different firmware images.
+
+.. _programming_usb:
+
+Quick programming of pre-compiled firmware images
+=================================================
+
+When you have the pre-compiled firmware images ready, you can directly program the images onto the Thingy:91 using the `nRF Connect Programmer`_ app, which is implemented as part of the `nRF Connect for Desktop`_.
+
+This is a direct USB method, making use of the MCUboot feature and the inbuilt serial recovery mode of Thingy:91.
```

Comment:
Not sure what we mean with "direct USB method".
Maybe:
When programming with nRF Connect Programmer, you connect the Thingy:91 directly to your PC through USB.
This method makes use of :doc:`mcuboot:index` and the inbuilt ...

Reply:
changed to : In this method, the Thingy:91 is connected directly to your PC through USB. This method makes use of ....